Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -58,7 +58,7 @@
     <Compile Remove="Common/ConfigurationStoreExceptionInterceptor.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.0" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
```

Comment:
Why bumping this dependency?

Reply:
Because there's a newer version?

Reply:
Unless we as a library need a new version, I think we should let that be up to the consumer.

Reply:
Fair point.